lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -968,7 +968,13 @@ public List<ViewExpression> getColumnMasks(SystemSecurityContext context, Catalo
                 .findFirst()
                 .stream()
                 .flatMap(Optional::stream)
-                .collect(toImmutableList());
+                .findFirst();
```

Comment:
There's already a `findFirst` here. This was deliberate - the SPI could return multiple masks, but File Based access control was still returning the first mask that matched, even if there were more.

```suggestion
                .findFirst()
                .flatMap(Function.identity());
```

Reply:
(Same applies to the connector-level one, of course)